frontend/src/scenes/cohorts/Cohort.tsx

```diff
@@ -1,203 +1,215 @@
 import React from 'react'
-import { CohortGroup } from './CohortGroup'
-import { cohortLogic } from './cohortLogic'
-import { Button, Card, Col, Divider, Input, Row } from 'antd'
-import { AimOutlined, ArrowLeftOutlined, InboxOutlined, UnorderedListOutlined } from '@ant-design/icons'
-import { useValues, useActions, BuiltLogic } from 'kea'
+import { useActions, useValues } from 'kea'
```

Comment:
Change is we actually moved `CohortV2.tsx` to here and dropped the legacy `Cohort.tsx` altogether.